Lib/test/libregrtest/main.py

```diff
@@ -769,7 +769,10 @@ def getloadavg(self):
             return self.win_load_tracker.getloadavg()
 
         if hasattr(os, 'getloadavg'):
-            return os.getloadavg()[0]
+            try:
+                return os.getloadavg()[0]
+            except OSError:
```

Comment:
The raised exception does not have any other information that `.args == ('Load averages are unobtainable',)`, so it's ok if we don't inspect it more to make sure we're catching only the issue at hand.